internal/tiltfile/updatesettings/update_settings.go

```diff
@@ -27,19 +28,28 @@ func (e Extension) OnStart(env *starkit.Environment) error {
 }
 
 func (e Extension) updateSettings(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
-	var maxParallelUpdates int
+	maxParallelUpdates := model.DefaultMaxParallelUpdates
```

Comment:
can you add a test for the case where someone calls update_settings twice?

```
update_settings(k8s_upsert_timeout_secs=60)
update_settings(max_parallel_updates=5)
```

I suspect this doesn't behave correctly.

Reply:
ha yeah i was trying to punt on that but you're right, it's too broken. i just made this function callable at max once.